demos/sslecho/main.c

```diff
@@ -0,0 +1,356 @@
+/*
+    Copyright 20xx-20yy The OpenSSL Project Authors. All Rights Reserved.
+
+    Licensed under the Apache License 2.0 (the "License").  You may not use
+    this file except in compliance with the License.  You can obtain a copy
+    in the file LICENSE in the source distribution or at
+    https://www.openssl.org/source/license.html
+ */
```

Comment:
Use the standard comment style, i.e.

````
/*
 *
 *
 */
````

See other files for examples

Reply:
Done.